source/draw/gpu/opengl/GraphicBuffer.ooc

```diff
@@ -46,8 +47,10 @@ GraphicBuffer: class {
 	length ::= this stride * this size y
 	nativeBuffer ::= this _nativeBuffer
 	handle ::= this _handle
+	identifier ::= this _identifier ?? this _handle
 
 	init: func (=_backend, =_nativeBuffer, =_handle, =_size, =_pixelStride, =_format, =_userData)
+	init: func ~identifier (=_backend, =_nativeBuffer, =_handle, =_size, =_pixelStride, =_format, =_userData, =_identifier)
```

Comment:
You should call this constructor from the other one with `_handle` as identifier